File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int r[]=new int[nums.length];
        int l[]=new int[nums.length];
        r[nums.length-1]=1;
        l[0]=1;
        for(int i=1;i<nums.length;i++){
            l[i]=l[i-1]*nums[i-1];
        }
        for(int i=nums.length-2;i>=0;i--){
            r[i]=r[i+1]*nums[i+1];
        }

        for(int i=0;i<nums.length;i++){
            nums[i]=l[i]*r[i];
        }
        return nums;
    }
}  

File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        for i in range(0,len(nums)-1,1):
            for j in range(i+1,len(nums),1):
                if nums[i]>nums[j]:
                    a=nums[i]
                    nums[i]=nums[j]
                    nums[j]=a

        print(nums)

        return nums
        